docs/integrations/parameter-types.md

```diff
@@ -12,7 +12,7 @@ See [here](https://xsoar.pan.dev/docs/integrations/yaml-file#configuration) for
 
 This type of parameter creates a checkbox in the integration configuration. When the checkbox is ticked, the value in
 the integration code is `True`, and `False` otherwise.
-The type number is: 0.
+The type number is: 8.
```

Comment:
Boolean is 8 and short text is 8?

Reply:
Boolean is 8, and Short Text is 0.
[Fixed](https://github.com/demisto/content-docs/pull/1393/commits/873142660357adc8e046244e73e716bc61a604f8).